payone-adapter/src/main/java/com/commercetools/sunrise/payment/payone/PayonePaymentServiceProviderImpl.java

```diff
@@ -0,0 +1,51 @@
+package com.commercetools.sunrise.payment.payone;
+
+import com.commercetools.sunrise.payment.domain.CreatePaymentDataProvider;
+import com.commercetools.sunrise.payment.domain.PaymentServiceProvider;
+import com.commercetools.sunrise.payment.utils.PaymentPropertiesLoadingHelper;
+import io.sphere.sdk.payments.Payment;
+import io.sphere.sdk.payments.PaymentMethodInfo;
+import io.sphere.sdk.payments.PaymentStatus;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiFunction;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ * Created by mgatz on 7/18/16.
+ */
+public class PayonePaymentServiceProviderImpl implements PaymentServiceProvider {
```

Comment:
removed the suffix

https://github.com/commercetools/commercetools-sunrise-java-payment/commit/8bf20a9d7da9255f19fdd0b8ba1c37f89b51d364